examples/system_tray_no_menu.rs

```diff
@@ -119,13 +128,21 @@ fn main() {
             // window size
             let window_inner_size = LogicalSize::new(200.0, 200.0);
             // create our window
-            let window = WindowBuilder::new()
+            let mut window_builder = WindowBuilder::new();
+            window_builder = window_builder
               // position our window centered with tray bounds
               .with_position(window_position_center_tray(&mut bounds, window_inner_size).position)
               .with_inner_size(window_inner_size)
-              .with_resizable(false)
-              .build(event_loop)
-              .unwrap();
+              // disallow resize
+              .with_resizable(false);
+
+            // skip taskbar on windows & linux
+            #[cfg(any(target_os = "linux", target_os = "windows"))]
+            {
```

Comment:
I believe you don't need `{}` here.

Reply:
getting'

![Screen Shot 2021-07-14 at 9 51 41 AM](https://user-images.githubusercontent.com/22237916/125633811-fcfd306a-2741-47af-be51-18feea8c1c6f.png)